jupyterhub-k8s-network-tools.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: jupyterhub-k8s-network-tools
+  version: 3.3.8
+  epoch: 0
+  description: Network diagnostic tools for use within a JupyterHub Kubernetes cluster
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - iptables
+
```

Comment:
Hey @sanghanan, is this commit missing some changes? This just looks like a package which includes iptables, runs a test, then it packages it up without any changes?

Reply:
Hey Mark! This is a weird package
Here's the thread for context - https://github.com/chainguard-dev/image-requests/issues/3789
We're essentially just packaging up wolfi base with iptables
If you see the [dockerfile](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/main/images/network-tools/Dockerfile), it contains just 2 lines: 
```dockerfile
FROM alpine:3.18
RUN apk add --no-cache iptables
```
If there's a better way to do this, I'd love to know!

Reply:
Thanks @sanghanan that helps. Added an inline suggestion above re: code comments